zoho.com.zmail_hosting.json

```diff
@@ -53,6 +53,13 @@
         "host": "%dkim_host%._domainkey",
         "data": "%dkim_data%",
         "ttl": 3600
+    },
+    {
+        "groupId":"zoho-dmarc",
+        "type": "TXT",
```

Comment:
DMARC is a bit tricky, as it actually affects the behavior of the whole domain.
I think the template should assure, that there is no DMARC record present already and also allow for removal of DMARC without dropping the whole template.

So I propose to add `txtConflictMatchingMode=All` to assure no duplicates and `essential=OnApply` to allow removal or change of the record.